README.md

```diff
@@ -4,10 +4,64 @@
 [![Coverage Status](https://coveralls.io/repos/github/andela/knights-bn-backend/badge.svg?branch=develop)](https://coveralls.io/github/andela/knights-bn-backend?branch=develop)
 [![Reviewed by Hound](https://img.shields.io/badge/Reviewed_by-Hound-8E64B0.svg)](https://houndci.com)
 
-Barefoot Nomad - Making company travel and accomodation easy and convinient.
-=======
-
+# Barefoot Nomad
 ## Vision
+##### Making company travel and accomodation easy and convinient.
+
+
+## Description
 Make company global travel and accommodation easy and convenient for the strong workforce of savvy members of staff, by leveraging the modern web.
 
----
+### Technonlogies
+- Environment : [Nodejs](https://nodejs.org/)
+- Package Manager : [NPM](https://www.npmjs.com)
+- Compiler : [Babel](https://babeljs.io/)
+- Testing : [Mocha and Chai](https://mochajs.org/)
+- Linting : [ESLint](https://eslint.org/)
+- Object-relational mapping (ORM) : [Sequelize](https://sequelize.org/)
+- Database : [postgresql](https://www.postgresql.org/)
+
+### Requirements and Installation steps
+
+#### You need the following to be able to run the application
+
+[Node](https://nodejs.org/en/download/) a runtime environment for JavaScript
+
+[Postman](https://www.getpostman.com/downloads/) to test the Api endpoints
+
+[Visual studio code](https://code.visualstudio.com/download) for editing and running the app
+
+[PostgreSQL](https://www.postgresql.org/download/) for Database
+
+### Installation
+
+##### A. Clone the project
+1. From your computer, open terminal 
+2. Run `https://github.com/andela/knights-bn-backend` OR [Download](https://github.com/andela/knights-bn-backend/archive/develop.zip) the project
+
+##### B. Setting up the environment
+1. Create a file and name it `.env` in root directory
+2. Find a file named `.env.example`
+3. Copy & paste all environmental variables from `.env.example` to `.env`
+4. Provide values to all environmental variables in `.env` file
+
+### Run commands
+
+Open terminal from your computer
+1. Run `npm install` to install all dependencies.
+2. Run `CREATE DATABASE databasename` to create your database.
+3. Run `npm run createTables` to create your database tables.
+4. Run `npm test` to test automatically if the is functioning properly as expected
+5. Run `npm start` to run the app in development environment. 
+6. copy URL`localhost:4000/api/v1/docs` to the browser for the API Documentation.
+
+7. copy URL`localhost:4000/api/v1/` to postman to start testing the endpoints.
+### API endpoints
+`- POST /auth/signup - Create user account`
```

Comment:
Add a link to the project swagger documentation or a postman collection

Reply:
it is not yet deployed!

Reply:
Alright. We shall add it when the app gets deployed.